pkgs/applications/misc/crumbs/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, gzip, ruby, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "crumbs";
+  version = "0.0.2";
+
+  src = fetchFromGitHub
+    { owner = "fasseg";
+      repo = "crumbs";
+      rev = "2ac7a909e66cff019acaa0566be721e3696f0bce";
```

```suggestion
      rev = "v${version}";
```
also, 0.0.3 seems to be out

---

@Thesola10 can you please address the 0.0.3 release comment?